host/linux/syscall.c

```diff
@@ -293,6 +293,27 @@ int oe_syscall_closedir_ocall(uint64_t dirp)
     return closedir((DIR*)dirp);
 }
 
+static void _stat_copy(const struct stat* st, struct oe_stat_t* buf)
+{
+    buf->st_dev = st->st_dev;
```

Comment:
It's an existing redundancy that got copied and pasted, but could you remove it?
```suggestion
```